tests/integration-tests/configs/additional_instance_types.yaml

```diff
@@ -0,0 +1,10 @@
+{%- import 'common.jinja2' as common with context -%}
+---
+test-suites:
+  schedulers:
+    test_slurm.py::test_slurm:
+      dimensions:
+        - regions: ["us-east-2"]
+          instances: [{{ instance_type_1|default("t2.micro") }}]
```

Comment:
another possibility could have been to not resolve instance_type_1 as a jinja variable but simply do a string replacement in the text file after jinja rendering is performed. The advantage is that you can treat this similarly to other instance names:
```
instances: ["instance_type_1", "c5.xlarge"]
# rather than
instances: ["{{ instance_type_1 }}", "c5.xlarge"]
```

Reply:
Changed to macro: `{{ common.instance("key") }}`